samples/highcharts/annotations-advanced/time-cycles/demo.js

```diff
@@ -0,0 +1,44 @@
+Highcharts.chart('container', {
+    chart: {
+        events: {
+            load: function () {
+                this.annotations.forEach(function (annotation) {
+                    annotation.setControlPointsVisibility(true);
+                    annotation.cpVisibility = true;
+                });
+            }
+        }
+    },
+
+    annotations: [{
+        type: 'timeCycles',
+        typeOptions: {
+            xAxis: 0,
+            yAxis: 0,
+            points: [{ x: 2 }, { x: 7 }]
+        }
+    }],
+
+    series: [{
+        data: [
```

Comment:
I think that this array may be in one line
Here you go: `data: [1, 2, 3, 5, 2, 3, 4, 5, 6, 7, 8, 3, 2, 4, 4, 4, 4, 3]`